### DISCUSSION TITLE
How to remove the entire rom.

### DISCUSSION THREAD

**Discussion by th3impasta** [cc-tweaked/CC-Tweaked#1335] [Q&A] [answered]
Hi!
I want to remove the entire rom of the computers. How do I do this?

I found a discussion that talked how to edit the rom and made a datapack that has just an empty rom but it doesn't remove the entire rom.

**Comment by SquidDev** [comment, marked as answer]
Unfortunately datapacks don't allow you to remove files - I believe vanilla has a similar issue where the easiest way to remove a recipe is to replace the recipe with an empty/malformed file.

I suspect the only way you can really do this is just by editing the original jar and deleting the files from there.

**Reply by th3impasta** [reply]
thanks! time to break out the ye olden quiltflower

**Reply by th3impasta** [reply]
or i could just fork the repository

**Reply by SquidDev** [reply]
You don't need to decompile anything! Just open the mod in 7zip (or any other zip viewer) and delete files directly!

But yes, forking also works :).

**Reply by th3impasta** [reply]
oh thx i forgot .jars are .zips in disguise

**Reply by th3impasta** [reply]
wait what librarys are included at the rom level?

**Comment by Lupus590** [comment]
Another way might be to use some kind of top level coroutine override.
[Unbios](https://gist.github.com/MCJack123/42bc69d3757226c966da752df80437dc) comes to mind as it also removes a bunch of stuff added but the ROM for you. You may still need to nil some globals and override the FS API so that the ROM folder can't be accessed.
